internal/storage/persistence/telemetry.go

```diff
@@ -180,6 +180,19 @@ func (w *wrapper) CreateWorkflow(ctx context.Context, workflow *types.Workflow)
 	return nil
 }
 
+func (w *wrapper) ReadWorkflow(ctx context.Context, dbID int) (*types.Workflow, error) {
```

Comment:
Doesn't need to happen here, but it would be nice to have a test for telemetry.go like we have on ingest.